src/components/InventoryV2/InventoryBatch.tsx

```diff
@@ -133,15 +133,21 @@ export default function InventoryBatch({ origin, species }: InventoryBatchProps)
       <PageHeaderWrapper nextElement={contentRef.current}>
         <BackToLink
           id='back'
-          name={`${strings.INVENTORY} / ${
-            origin === 'Species'
-              ? strings.formatString(strings.BATCHES_OF, getSpeciesLabel())
-              : strings.formatString(strings.BATCHES_AT, getNurseryLabel())
-          } `}
+          name={`${strings.INVENTORY}${
+            origin !== 'Inventory'
```

Comment:
When linked from the batches list page, the breadcrumb should not have a species or nursery